sources/packages/api/src-sql/SINValidations/Add-column-student-id.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.sin_validations
+ADD
+    COLUMN IF NOT EXISTS student_id INT REFERENCES sims.students(id);
+
+COMMENT ON COLUMN sims.sin_validations.student_id IS 'Student to whom the SIN validation record belongs to.';
```

Comment:
Can you please indicate in the comment that this is a foreign key reference to students table just like you have done in the comment for user_id.

Reply:
IMO when I mention foreign key in the comment it would be just rephrasing the verbose. My intention here was to provide meta-data/business context to the column. @andrewsignori-aot @ann-aot please let me know your thoughts as well.

Reply:
Sure, resolving.